armbian/build
# [ mvebu ] Switch fake-hwclock to hardware RTC AR-159

* remove fake-hwclock dpkg
* tweak /lib/udev/hwclock-set to be executed even if systemd present
* move helios4 bsp file under mvebu folder for tidiness

# how to test

1. boot you system with network
2. let the system ntp sync
3. check that both system time and rtc time are updated

$> timedatectl status
Local time: Thu 2020-05-07 14:46:48 +08
Universal time: Thu 2020-05-07 06:46:48 UTC
RTC time: Thu 2020-05-07 06:46:48
[...]

4. poweroff system and disconnect PSU
5. wait 10 min
6. **remove network cable**
7. start system without network (you don't want chrony to sync the time)
8. check system is still at current time

$> timedatectl status
Local time: Thu 2020-05-07 14:58:30 +08
Universal time: Thu 2020-05-07 06:58:30 UTC
RTC time: Thu 2020-05-07 06:58:30
